transitions/index.js

```diff
@@ -15,6 +15,8 @@ const _ = require('underscore'),
       PROGRESS_REPORT_INTERVAL = 500, // items
       transitions = {};
 
+let feed;
```

Comment:
`feed` doesn't really mean anything to me. However, this is probably because I'm not familiar with the follow.feed function. Maybe `changesFeed` would make more sense?

Anyway, it's not hard to work out what it does by just looking for `feed =` in the codebase. I'll leave it to you if you want to consider a rename.